docs/src/main/sphinx/release/release-363.md

```diff
@@ -0,0 +1,63 @@
+# Release 363 (XX Oct 2021)
+
+## General
+
+* Reduce query memory usage when spill occurs. ({issue}`9270`)
+* Reduce join memory usage. ({issue}`9327`)
+* Reduce query memory usage during join unspilling. ({issue}`9275`)
+* Fix potential data loss in query results when clients retry requests to the coordinator. ({issue}`9453`)
+* Fix decimal arithmetic to not produce negative zeros. Previously comparing two zero decimal
+  values might have returned `false` because of dual zero representation. ({issue}`8985`)
+* Fix `SHOW ROLES` failure when there are no roles to display. ({issue}`9319`)
+* Fix `EXPLAIN ANALYZE` to show estimates. ({issue}`9396`)
+
+## Python client
+
+* Fix column type reported in `cursor.description` for `time with time zone` column. ({issue}`9460`)
+
+## BigQuery connector
+
+* Fix failure for queries where predicate on `geography`, `array` or `struct` column is pushed down to BigQuery. ({issue}`9391`)
+
+## Cassandra connector
+
+* Add support for Cassandra TUPLE type. ({issue}`8570`)
+
+## Elasticsearch connector
+
+* Add support for `scaled_float` type. ({issue}`9358`)
+
+## Hive connector
+
+* Improve query performance when reading Parquet data with predicate on a `decimal` column. ({issue}`9338`)
+* Fix insert overwrite operation on S3 backing storage. ({issue}`9234`)
```

Comment:
Is it "Fix" or "Add support for"? @aczajkowski ?

Reply:
Also, if it's "fix", what was the issue? Query failure? Data loss? Etc.

Reply:
Yes it's fix. There was no inf. in docs that we don't support overwrite in S3 compatible storages. And it was explicitly denied by check in code. 
It looks that check was added to prevent wrong behaviour in such cases. This PR fixed behaviour and removed check in code denying this operation.

Reply:
And behaviour before fix was not removing old data. And effectively we got old and new records in table.

Reply:
This actually looks redundant with this one: https://github.com/trinodb/trino/pull/9495#discussion_r722453351